kyuubi-hive-beeline/pom.xml

```diff
@@ -78,6 +78,17 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <exclusions>
```

Comment:
The `commons-io` does not has transitive dependencies, we don't need this exclusion